### ISSUE TITLE
Update dependencies to pass npm audit, according to high risk issues

### ISSUE THREAD

**Issue by joeherold** [Urigo/graphql-scalars#44]
![image](https://user-images.githubusercontent.com/3911545/61651457-35506f80-acb6-11e9-8de9-b79343484ac7.png)

**Comment by lookfirst** [comment]
Also...

```
npm install --save graphql-scalars

npm WARN @graphql-modules/di@0.7.5 requires a peer of reflect-metadata@^0.1.12 but none is installed. You must install peer dependencies yourself.
```

**Comment by joeherold** [comment]
I have no issues with peer dependencies

**Comment by joeherold** [comment]
@lookfirst I switched to [@saeris/graphql-scalars](https://www.npmjs.com/package/@saeris/graphql-scalars)

**Comment by lookfirst** [comment]
@joeherold No typescript and also looks dead now... https://github.com/Saeris/graphql-scalars/issues/23

**Comment by Urigo** [comment]
@joeherold I'm sorry, we've just took over the library and I forgot to activate `renovatebot` on it.
I did it now, we will upgrade it all and release a new version in the next hour.

also @lookfirst , @ardatan will move `graphql-modules` to be a peer dependency before the new release.

@joeherold I'm sorry for the late reply, this is unusual for us (The Guild), we are doing daily status for all of our libraries and somehow missed this specific issue.
I don't think you have a reason to switch to another package

**Comment by joeherold** [comment]
@the-guild-org no stress :-)

**Comment by ardatan** [comment]
Fixed in 0.6.7!

**Comment by ardatan** [comment]
Fixed in the latest version!
